live_flights_on_map.md

```diff
@@ -0,0 +1,125 @@
+## Documentation for live flight details on a map
+
+##### 1. Map Creation
+
+Firstly, we have to insert OpenLayers css into an HTML file:
+
+```html
+<link rel="stylesheet"href="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.1.1/css/ol.css"type="text/css">
+```
+
+Then we have to insert OpenLayers js file:
+
+```html
+<script src="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.1.1/build/ol.js"></script>
+```
+
+Inside `body` of the html file, we have to create a `map` \<div>:
+
+```html
+<div id="map" class="map"></div>
+``` 
+
+In the above `map` \<div>, our map will reside.
+
+Next, we will add an OSM map into the map:
```

Comment:
@arnabdutta73 can you add more details about the functions you have used here `ol.Map`, `ol.View`, `ol.layer.Tile` and `ol.fromLonLat`?  This explanation will help the reader to understand the code and these functions much easier without switching the context.

Reply:
Done